api/turing/service/ensembling_job_service.go

```diff
@@ -21,8 +21,8 @@ type EnsemblingJobFindByIDOptions struct {
 // EnsemblingJobListOptions holds query parameters for EnsemblersService.List method.
 type EnsemblingJobListOptions struct {
 	PaginationOptions
-	ProjectID *models.ID
-	Status    *models.Status `validate:"oneof=pending running terminating terminated completed failed failed_submission failed_building"` //nolint
+	ProjectID *models.ID `schema:"project_id" validate:"required"`
```

Comment:
How is the `schema:` tag used?

Reply:
The schema is used by parsing the query params into here: https://github.com/gojek/turing/pull/63/files#diff-8d901405e0fd6037fb5bb3c53aade949ea58f83a31c57b302a6983eb3c8eee60R96

I saw that @romanwozniak used this technique to parse the query params into the struct.